helm/values/elasticsearch.yaml

```diff
@@ -0,0 +1,3 @@
+---
+
+clusterName: "chime-dev"
```

Comment:
The official cluster name I think is chime-bus, (it's back up in case prod get hits by a bus...)

Reply:
ok Great. I'll make a change.

Reply:
```suggestion
clusterName: "chime-bus"
```